docs/tutorials/denom-allolist-proposal.md

```diff
@@ -0,0 +1,40 @@
+# Currency Allow list Proposals
+
+Community members, or representatives of projects/protocols are encouraged to submit proposals for allowing currencies to be used in the marketplace.
+
+This document provides instructions for submitting a currency allowlist proposal using the [command-line interface (CLI)](../ledger/infrastructure/interfaces.html#command-line-interface).
```

Comment:
We use `.md` in the markdown documentation. The links are automatically updated by vuepress.

```suggestion
This document provides instructions for submitting a currency allowlist proposal using the [command-line interface (CLI)](../ledger/infrastructure/interfaces.md#command-line-interface).
```